CHANGELOG.md

```diff
@@ -180,6 +180,8 @@ This release includes a number of new features and bugfixes like a new config su
 
 - tests/aws.sh: add test for Fedora 24 ([#2340](https://github.com/coreos/rkt/pull/2340)).
 
+- Add a new subcommand `rkt stop` ([#1959](https://github.com/coreos/rkt/pull/1959)).
```

Comment:
There is now a script to generate the changelog when doing a release, so it is no longer necessary to add an entry for each PR.